kiali/README.md

```diff
@@ -40,22 +40,38 @@ An alternative can be a parallel installation of the upstream chart offering all
 
 1. Run the Helm upgrade command, which installs the chart if not present yet.
     ```bash
-    helm upgrade --install --create-namespace -n ${KYMA_KIALI_NS} ${HELM_RELEASE_NAME} kiali/kiali-operator --set cr.spec.auth.strategy=anonymous -f https://raw.githubusercontent.com/kyma-project/examples/main/kiali/values.yaml
+    helm upgrade --install --create-namespace -n ${KYMA_KIALI_NS} ${HELM_RELEASE_NAME} kiali/kiali-operator -f https://raw.githubusercontent.com/kyma-project/examples/main/kiali/values.yaml
     ```
 
 You can either use the `[values.yaml](./values.yaml)` provided in this `kiali` folder, which contains customized settings deviating from the default settings, or create your own `values.yaml` file.
 
 ### Verify the installation
 
-1. You should see the `kiali-operator` and `kiali` Pod coming up in the Namespace. All Pods must eventually be in a "Running" state.
-1. Browse the Kiali dashboard. The following command exposes the dashboard on `http://localhost:20001`:
-   ```bash
-   kubectl -n ${KYMA_KIALI_NS} port-forward svc/kiali-server 20001
-   ```
+You should see the `kiali-operator` and `kiali-server` Pod coming up in the Namespace. All Pods must eventually be in a "Running" state.
+
+### Access Kiali
+
+To access Kiali, you can either use kubectl port forwarding, or expose it using the Kyma ingress gateway.
+
+* To access Kiali using port forwarding, run:
+  ```bash
+  kubectl -n ${KYMA_KIALI_NS} port-forward svc/kiali-server 20001
+  ```
+
+  Open Kiali in your browser under [http://localhost:20001](http://localhost:20001).
+
+* To expose Kiali using the Kyma ingress gateway, create an APIRule:
+  ```bash
+  kubectl -n ${KYMA_KIALI_NS} apply -f https://raw.githubusercontent.com/kyma-project/examples/main/kiali/apirule.yaml
+  ```
+  Get the public URL of your Kiali server:
+  ```bash
+  kubectl -n ${KYMA_KIALI_NS} get virtualservices.networking.istio.io -ojsonpath='{.items[*].spec.hosts[*]}'
```

Comment:
And the URL is not available in the APIRule which we applied before? Like to avoid that people need to know both resources if possible